public/js/editors/libraries.js

```diff
@@ -107,6 +107,13 @@ var libraries = [
   },
   {
     'url': [
+      '//himera-emh.herokuapp.com/js/repl.js'
```

Comment:
I don't think this is needed in the libraries dropdown...though I'm not totally sure what the intention is here.